src-tauri/src/main.rs

```diff
@@ -627,6 +628,43 @@ fn log_web_message(level: String, message: Vec<String>) {
     }
 }
 
+#[tauri::command]
+async fn reset_settings<'r>(
+    _window: tauri::Window,
+    state: tauri::State<'r, UniverseAppState>,
+    app: tauri::AppHandle
+) -> Result<(), String> {
+    let app_config_dir = app.path_resolver().app_config_dir();
+    let app_cache_dir = app.path_resolver().app_cache_dir();
+    let app_data_dir = app.path_resolver().app_data_dir();
+    let app_local_data_dir = app.path_resolver().app_local_data_dir();
+
+    let dirs_to_remove = vec![app_config_dir, app_cache_dir, app_data_dir, app_local_data_dir];
+    let missing_dirs: Vec<String> = dirs_to_remove.iter().filter(|dir| dir.is_none()).map(|dir| dir.clone().unwrap().to_str().unwrap().to_string()).collect();
+
+    if missing_dirs.clone().len() > 0 {
+        error!("Could not get app directories for {:?}", missing_dirs);
+        return Err("Could not get app directories".to_string());
+    }
+
+    dirs_to_remove.iter().for_each(|dir| {
+        // check if dir exists
+        if dir.clone().unwrap().exists() {
+            info!(target: LOG_TARGET, "[reset_settings] Removing {:?} directory", dir);
+            remove_dir_all(dir.clone().unwrap()).unwrap();
+        }
+    });
+
+    state.shutdown.clone().trigger();
```

Comment:
This needs to happen before removing the dirs above. I suggest moving this to line 637